test/test.cnf

```diff
@@ -49,14 +49,14 @@ emailAddress		= optional
 
 ####################################################################
 [ req ]
-default_bits		= 2048
+default_bits		= 2048 # TODO avoid using this, for more efficient tests
```

Comment:
Please drop this. This TODO brings no value.

Reply:
This (and the one mentioned below) was a leftover that I forgot to handle.
Solved as I wrote above.